syz-cluster/pkg/api/api.go

```diff
@@ -154,86 +155,76 @@ type BuildInfo struct {
 }
 
 // Let them stay here until we find a better place.
+// The list is ordered by decreasing importance.
 var DefaultTrees = []*Tree{
 	{
-		Name:         `torvalds`,
-		URL:          `https://kernel.googlesource.com/pub/scm/linux/kernel/git/torvalds/linux`,
-		Branch:       `master`,
-		Priority:     0,
-		EmailLists:   []string{},
-		KernelConfig: `upstream-apparmor-kasan.config`,
-		FuzzConfig:   `all`,
+		Name:       `bpf-next`,
```

Comment:
Does `bpf` has higher priority than `net` from your point of view?
Or they are equivalent and this order makes sure `-next` always goes first?

Reply:
I'd say if bpf and net are both Cc'd, then it's more likely that netdev (as more generic list) was just mentioned and the bpf (as a more specific list) is the target recipient.

In the end, I think I'll end up calling `get_maintainer.pl` to get an ordered list of subsystems/trees, in which case the decision will be driven by the code changes. Fow now it all is just an approximation.

Reply:
Got it. Thanks.